test/integration/project_test.go

```diff
@@ -106,12 +106,14 @@ func TestProjectCreate_ApplicationDoesntExist_FailAndReturnErrorMessage(t *testi
 
 func TestProjectCreate_ApplicationExists_CreateProjectSuccessfully(t *testing.T) {
 
-	err, _ := executeCommand(
+	err, outBuffer := executeCommand(
 		t, "project", "create", flag(params.FormatFlag),
 		printer.FormatJSON, flag(params.ProjectName), projectNameRandom,
 		flag(params.ApplicationName), "my-application",
 	)
-
+	createdProject := wrappers.ProjectResponseModel{}
+	unmarshall(t, outBuffer, &createdProject, "Reading project create response JSON should pass")
+	defer deleteProject(t, createdProject.ID)
 	assert.NilError(t, err)
```

Comment:
please add assert as you did in the other test

Reply:
done